pkg/ibmcsidriver/server.go

```diff
@@ -137,6 +158,35 @@ func (s *nonBlockingGRPCServer) Setup(endpoint string, ids csi.IdentityServer, c
 	return listener, nil
 }
 
+func (f *realFileOps) Chown(name string, uid, gid int) error {
+	return os.Chown(name, uid, gid)
+}
+
+func (f *realFileOps) Chmod(name string, mode os.FileMode) error {
+	return os.Chmod(name, mode)
+}
+
+func setupSidecar(addr string, fileOps FileOps) error {
+	groupSt := os.Getenv("SIDECAR_GROUP_ID")
+	group, err := strconv.Atoi(groupSt)
```

Comment:
expectation here is to use correct value from deployment else it should be `0` root

Reply:
done